property-scraper.js

```diff
@@ -1,8 +1,10 @@
 'use strict';
-var request = require("request"),
+const request = require("request"),
     cheerio = require("cheerio"),
     parser = require('parse-address'),
-    assert = require('assert');
+    assert = require('assert'),
+    csv = require(`csv-parser`),
```

Comment:
only use backticks for genuine template strings. otherwise prefer single quotes for strings.